test/README.md

```diff
@@ -34,6 +39,14 @@ for example:
 docker run -e PTP_TEST_MODE=OC -e ENABLE_TEST_CASE=reboot -v /home/usr/.kube/config.3nodes:/tmp/config:Z -v .:/output:Z quay.io/redhat-cne/ptp-operator-test:latest
 ```
 
+## Run SOAK test
+
+Soak tests are configurable through the test configuration file as YAML file stored at location `test/conformance/ptptestconfig.yaml`. These tests run in parallel.
```

Comment:
how do you override this configuration when you run vai docker run or makefile, can you pass the configuration file to override ?

Reply:
I have the impression that the users of these tests would copy the entire codebase and change it manually and running Makefile or docker is not what they would use, please correct me here.